diff --git a/node_modules/whisper.rn/whisper-rn.podspec b/node_modules/whisper.rn/whisper-rn.podspec
index b85bb4f..7cafca1 100644
--- a/node_modules/whisper.rn/whisper-rn.podspec
+++ b/node_modules/whisper.rn/whisper-rn.podspec
@@ -28,7 +28,7 @@ Pod::Spec.new do |s|
   s.license      = package["license"]
   s.authors      = package["author"]
 
-  s.platforms    = { :ios => "11.0", :tvos => "11.0" }
+  s.platforms    = { :ios => "11.0" }
   s.source       = { :git => "https://github.com/mybigday/whisper.rn.git", :tag => "#{s.version}" }
 
   s.requires_arc = true
